tests/kafkatest/services/verifiable_consumer.py

```diff
@@ -123,7 +123,7 @@ class VerifiableConsumer(BackgroundThreadService):
     logs = {
         "verifiable_consumer_stdout": {
             "path": STDOUT_CAPTURE,
-            "collect_default": False},
+            "collect_default": True},
```

Comment:
It was intentional since it helps debugging quite a bit. The verifiable consumer only logs the start and end offsets of the batches of records it consumes, so the size is pretty reasonable. At the same time, it's probably unnecessary since I already increased offset logging in that other ticket.